velox/connectors/Connector.h

```diff
@@ -276,15 +276,27 @@ class ConnectorFactory {
   const std::string name_;
 };
 
+/// Adds a factory for creating connectors to the registry using connector name
```

Comment:
It isn't clear what connector name is, since it isn't a parameter to the function. Please can you explain.